scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -281,17 +281,14 @@ SG_ MIN_TEMP_IDX : 18|4@1+ (1,0) [0|15] "" DEBUG
 SG_ MAX_TEMP_SEGMENT : 22|3@1+ (1,0) [0|7] "" DEBUG
 SG_ MAX_TEMP_IDX : 25|4@1+ (1,0) [0|7] "" DEBUG
 
-BO_ 117 BMS_TRACTIVE_SYSTEM: 5 BMS
+BO_ 117 BMS_TRACTIVE_SYSTEM: 9 BMS
```

Comment:
is this not giving errors during CAN code gen? Max payload size for a CAN message should be 8 bytes. I suggest splitting this into two messages for now